.github/actions/setup-elasticsearch/action.yml

```diff
@@ -4,14 +4,6 @@ description: Sets up Elasticsearch
 runs:
   using: composite
   steps:
-    - name: Configure sysctl limits
```

Comment:
Why don't we need this, while the documentation says otherwise? https://github.com/elastic/elastic-github-actions/tree/master/elasticsearch

Reply:
As far as I can tell, this is already included in the action that's called: https://github.com/elastic/elastic-github-actions/blob/master/.github/workflows/elasticsearch.yml

Reply:
But you are using this https://github.com/elastic/elastic-github-actions/blob/master/elasticsearch/action.yml action, not the workflow you linked, which is totally fine and it should be like that.

Reply:
Hmm. It's odd that it works this way too. Either way, I'll restore the sysctl limits step.